model/Core/Vocabularies/RelationshipType.md

```diff
@@ -52,10 +52,10 @@ name completes the sentence:
 - hasAddedFile: Every `to` Element is a file added to the `from` Element (`from` hasAddedFile `to`).
 - hasAssessmentFor: Relates a `from` Vulnerability and each `to` Element with a security assessment. To be used with `VulnAssessmentRelationship` types.
 - hasAssociatedVulnerability: Used to associate a `from` Artifact with each `to` Vulnerability.
-- hasConcludedLicense: The `from` SoftwareArtifact is concluded by the SPDX data creator to be governed by each `to` license.
+- hasConcludedLicense: The `from` SoftwareArtifact is concluded by the SPDX data creator to be governed by each `to` AnyLicenseInfo.
```

Comment:
```suggestion
- hasConcludedLicense: The `from` SoftwareArtifact is concluded by the SPDX data creator to be governed by the `to` AnyLicenseInfo.
```

Reply:
For that change, please refer back to the original PR #1122.

This PR will minimally focus only on the type of license and nothing else.